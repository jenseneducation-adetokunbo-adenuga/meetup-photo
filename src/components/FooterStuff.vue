<template>
	<div>
 <v-footer app></v-footer>
	</div>
</template>

<script>
	export default {
		
	}
</script>

<style lang="scss" scoped>

</style>