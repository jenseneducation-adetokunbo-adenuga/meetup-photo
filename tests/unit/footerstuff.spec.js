import FooterStuff from "@/components/FooterStuff";
import { createLocalVue, mount } from "@vue/test-utils";
import Vuetify from "vuetify";

describe("FooterStuff.vue", () => {
  const localVue = createLocalVue();
  let vuetify;

  beforeEach(() => {
    vuetify = new Vuetify();
  });

  it("should have a custom title and match snapshot", () => {
    const wrapper = mount(FooterStuff, {
      localVue,
      vuetify,
      propsData: { title: "Foobar" },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
