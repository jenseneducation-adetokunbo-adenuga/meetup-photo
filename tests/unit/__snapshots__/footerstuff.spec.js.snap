// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterStuff.vue should have a custom title and match snapshot 1`] = `
<div title="Foobar">
  <v-footer dark="" padless="">
    <v-card flat="" tile="" class="black lighten-1 white--text text-center">
      <v-card-text>
        <v-btn icon="" class="mx-4 white--text">
          <v-icon size="24px">
            mdi-facebook
          </v-icon>
        </v-btn>
        <v-btn icon="" class="mx-4 white--text">
          <v-icon size="24px">
            mdi-twitter
          </v-icon>
        </v-btn>
        <v-btn icon="" class="mx-4 white--text">
          <v-icon size="24px">
            mdi-linkedin
          </v-icon>
        </v-btn>
        <v-btn icon="" class="mx-4 white--text">
          <v-icon size="24px">
            mdi-instagram
          </v-icon>
        </v-btn>
      </v-card-text>
      <v-card-text class="white--text pt-0">
        Phasellus feugiat arcu sapien, et iaculis ipsum elementum sit amet.
        Mauris cursus commodo interdum. Praesent ut risus eget metus luctus
        accumsan id ultrices nunc. Sed at orci sed massa consectetur dignissim
        a sit amet dui. Duis commodo vitae velit et faucibus. Morbi vehicula
        lacinia malesuada. Nulla placerat augue vel ipsum ultrices, cursus
        iaculis dui sollicitudin. Vestibulum eu ipsum vel diam elementum
        tempor vel ut orci. Orci varius natoque penatibus et.
      </v-card-text>
      <v-divider></v-divider>
      <v-card-text class="white--text">
        2020 â€” <strong>A.D King productions</strong></v-card-text>
    </v-card>
  </v-footer>
</div>
`;
